var/spack/repos/builtin/packages/r-gtools/package.py

```diff
@@ -37,12 +38,12 @@ class RGtools(RPackage):
       match ('smartbind')
     - generate significance stars from p-values ('stars.pval')
     - convert characters to/from ASCII codes.
```

Comment:
Spack removes newline characters from the description when running `spack info`, so this won't format well. Maybe add semicolons at the end of each bullet point?

Reply:
I added semicolons to the end of each list item as well as converted the '-'s to '[#]s'.